crates/red_knot/src/main.rs

```diff
@@ -27,7 +27,7 @@ mod verbosity;
 #[command(
     author,
     name = "red-knot",
-    about = "An experimental multifile analysis backend for Ruff"
+    about = "A Python type checker, written in Rust"
```

Comment:
```suggestion
    about = "An extremely fast Python type checker."
```

Reply:
(Let's see how it turns out I guess, but good to aspire to this anyway 😆)